lib/tapioca/cli.rb

```diff
@@ -97,12 +142,24 @@ def dsl(*constants)
       banner: "gem:level [gem:level ...]",
       desc: "Overrides for typed sigils for generated gem RBIs"
     def generate(*gems)
+      deprecation_message = <<~MSG
+        DEPRECATION: The `generate` command will be removed in a future release.
+
+        Start using the `gem` command with gem names or the `--all` flag instead.
```

Comment:
We could even use the args to suggest the exact replacement:

```rb
"Use `tapioca gem #{args.join(" ")}` instead"
```

Reply:
This is a good idea, I'll implement this.